clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -2735,6 +2735,41 @@ TEST(TransferTest, ResultObjectLocationForDefaultInitExpr) {
       });
 }
 
+// This test ensures that CXXOperatorCallExpr returning prvalues are correctly
+// handled by the transfer functions, especially that `getResultObjectLocation`
+// correctly returns a storage location for those.
+TEST(TransferTest, ResultObjectLocationForCXXOperatorCallExpr) {
+  std::string Code = R"(
+    struct A {
+      virtual ~A() = default;
```

Comment:
Any reason this needs to have a virtual destructor? This looks as if it's just copy-pasted from the ResultObjectLocation test above? Would suggest deleting this.